src/components/AddBook/AddBookElements.js

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components';
+
+export const FormWrapper = styled.div`
+    width: 100%;
+    max-width: 1300px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    margin: 2.813rem;
+
+    
+`;
+export const FormContainer = styled.form`
```

Comment:
- [ ] It's better to create a CSS file that contains all styling codes, and this message applies to all the files.

Reply:
Hi @dasileker,

Do you mean I can not use styled-components which is like sass in my react project?

Reply:
hello @micheaol yes you can create a CSS file and import the CSS to your js file.

Reply:
Do you mean I can not use styled components in react app?